server/__tests__/lobbies.test.js

```diff
@@ -0,0 +1,53 @@
+import { addLobby, LOBBY_LIFESPAN } from "../lobbies";
+
+// Mock persistence so DB doesn't change during tests
+jest.mock("../persistence", () => ({
+  save: jest.fn(),
+  load: jest.fn(() => []),
+}));
+
+describe("Lobby actions", () => {
+  describe("Lobby expiration", () => {
+    beforeEach(() => {
+      jest.useFakeTimers();
+    });
+
+    it("should be deleted after 15min of inactivity", () => {
+      const lobbyData = {
+        id: "rbbfJ5t3Zzpcj0TZxmYsM",
+        players: ["2VDBVQ-6MD2F1rWJXdgyBodBp0YjAY88"],
+        code: "7025",
+      };
+      addLobby(lobbyData);
+
+      const spliceFn = jest.spyOn(Array.prototype, "splice");
+      jest.advanceTimersByTime(LOBBY_LIFESPAN);
+      expect(spliceFn).toHaveBeenCalled();
```

Comment:
You can check here that `lobbies` is empty instead.

Reply:
`splice` could be called by any other method so this check isn't really reliable.

Reply:
Yes, I agree. My original idea was to spy on `deleteLobby()`, but I couldn't make it work because jest can't spy on functions of the same import. Thanks for the suggestions!